engine/src/main/resources/swagger/cromwell.yaml

```diff
@@ -716,6 +756,209 @@ definitions:
         type: string
         description: The status of the workflow
         example: Submitted
+  WorkflowDescription:
+    required:
+      - valid
+    properties:
+      valid:
+        type: boolean
+        description: Whether the workflow and inputs as submitted passes validation
+      errors:
+        type: array
+        items:
+          type: string
+        example:
+          - "The 'errors' field will be filled if 'valid' is false"
+          - "We might also provide warnings to a 'valid' workflow here"
+          - "Otherwise, 'errors' will be empty or unspecified"
+        description: The set of validation failure messages
+      name:
```

Comment:
Is this an optional or nullable field ? I'm thinking of the case where the workflow is so broken that it doesn't have a name (or a valid one)

Reply:
Correct - only the `valid` field is listed at the top in the `required` section because it's the only field that's always required. 

As a side comment, OpenAPI 3 lets you be more expressive with `oneOf` format options where we could have separate "if valid" and "if invalid" formats and say that the result has to be "one of" the two..., but since we're still using Swagger 2, that kind of "name will only appear if `valid` is `true`" logic has to be informal